src/ViewHelpers/ArgumentViewHelper.php

```diff
@@ -34,6 +34,9 @@
  * in any of its arguments is not possible (e. g. you can't define an argument name
  * by using a variable).
  *
+ * Currently rendering of specific sections will not validate argument constraints. 
```

Comment:
Avoid using "currently" - it implies that something might change.. but you don't know :)

Keep it clean:

`Rendering of specific sections will not validate argument constraints. 
Comment`